GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/MountTests.cs

```diff
@@ -210,6 +210,21 @@ public void MountRegeneratesAlternatesFileWhenMissingFromDisk()
             alternatesFilePath.ShouldBeAFile(this.fileSystem).WithContents(objectsRoot);
         }
 
+        [TestCase]
+        public void MountCanProcessSavedBackgroundQueueTasks()
+        {
+            this.Enlistment.UnmountGVFS();
+
+            string deleteFilePath = Path.Combine("Test_EPF_WorkingDirectoryTests", "1", "2", "3", "4", "ReadDeepProjectedFile.cpp");
+            string deleteDirPath = Path.Combine("Test_EPF_WorkingDirectoryTests", "1", "2", "3", "4");
+            this.fileSystem.WriteAllText(
+                Path.Combine(this.Enlistment.EnlistmentRoot, ".gvfs", "databases", "BackgroundGitOperations.dat"),
+                "A 1\03\0{deleteFilePath}\0\r\nA 2\011\0{deleteDirPath}\0\r\n");

```

Comment:
`"A 1\03\0{deleteFilePath}\0\r\nA 2\011\0{deleteDirPath}\0\r\n"`

This is missing the `$` and so `{deleteFilePath}` and `{deleteDirPath}` are not being substituted.

The fact that the test didn't fail makes me think it is worth implementing your suggestion of calling `git status` as well at the end of this test.  That would have caught that ModifiedPaths.dat was not updated as expected by VFS4G. when it ran the background tasks.

Also, minor style suggestion here:  It might be a bit easier to understand what's being written to "BackgroundGitOperations.dat" if each line is split out.  Something like:

```
string persistedDeleteFileTask = $"A 1\03\0{deleteFilePath}\0";
string persistedDeleteDirectoryTask = $"A 2\011\0{deleteDirPath}\0";

this.fileSystem.WriteAllText(
                Path.Combine(this.Enlistment.EnlistmentRoot, ".gvfs", "databases", "BackgroundGitOperations.dat"),
                $"{persistedDeleteFileTask}\r\n{persistedDeleteDirectoryTask}\r\n");
```